### PR TITLE
UI: indicate edit permission in status bar

### PR DESCRIPTION
**Pull request by lpranam**
update edit permission indicator based on WOPI parameters received

Change-Id: I83a743f8aff646e357ce1a0e73abf9d3db99280e

* Target version: master 

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/src/control/Control.StatusBar.js [hunk 1]
```diff
@@ -377,12 +377,19 @@ class StatusBar extends JSDialog.Toolbar {
...
+			permissionContainer.appendChild(getPermissionModeElements(isReadOnlyMode, this.map['wopi'].UserCanWrite));
+		}
+
 		this.builder.updateWidget(this.parentContainer, {
 			id: 'PermissionMode',
 			type: 'htmlcontent',
 			htmlId: 'permissionmode',
 			isReadOnlyMode: isReadOnlyMode,
-			canUserWrite: !app.isReadOnly()
+			canUserWrite: this.map['wopi'].UserCanWrite
```
Comment (caolanm) [inline browser/src/control/Control.StatusBar.js:392]: I wonder if this is ok to do for the android/iOS case? I presume there that this.map['wopi'] would be null, maybe @Minion3665  knows. Would is be safer to use something like:

 this.map['wopi'] ? this.map['wopi'].UserCanWrite : !app.isReadOnly()

Comment (Minion3665) [reply browser/src/control/Control.StatusBar.js:392]: A bit worse than that I'm afraid - in the mobile case `this.map['wopi']` *isn't* null but `L.Map.THIS.wopi.UserCanWrite` is always false.

here's some stuff from my chrome debug console...
```
$ L.Map.THIS["wopi"]
NewClass {_map: NewClass, _initHooksCalled: true}

$ L.Map.THIS.wopi.UserCanWrite
false
```

**Review by Minion3665 — COMMENTED** [review]
Thanks for correcting the mobile issues - this is all good from my end now :)

**Review by caolanm — APPROVED** [review]
ok, lets merge then

**Pull request merged**